docs/get-started/_index.md

```diff
@@ -88,100 +83,42 @@ motor->set_power(0.4);
 {{% /tab %}}
 {{< /tabs >}}
 
-The standardized nature of Viam's resource APIs means that when you build machines, **you can swap out components such as one motor for another motor of a different brand without changing any code**.
-The only change needed is to your machine configuration.
-
-However, if you need more custom behaviour, you can [extend these APIs to suit your own needs](/registry/advanced/create-subtype/).
-
-### Connect from anywhere
-
-{{<gif webm_src="/fleet/mobile-app-control.webm" mp4_src="/fleet/mobile-app-control.mp4" alt="Using the control interface under the locations tab on the Viam mobile app" class="alignright" max-width="300px">}}
-
-Viam uses WebRTC and gRPC for **secure peer-to-peer communications across network boundaries**.
-
-In factory settings or remote environments, you can operate and monitor your machines over a local area network (LAN).
-Any monitoring or control applications can **connect directly to `viam-server` without internet connection** or run on the machines themselves.
-
-If your machines are connected to the internet, you can operate and monitor them **from anywhere in the world**.
-
-You can use the Viam app and the Viam mobile app, to control and monitor your machines securely.
-Or you can use Viam's SDKs to build your own apps.
-
-### Support
-
-The `viam-server` executable runs on **any computer that runs on 64-bit Linux OS, macOS, or Windows Subsystem for Linux**.
-Viam also supports microcontrollers with [`viam-micro-server`](/installation/#install-viam-micro-server).
-
-Here are some of the systems Viam runs on:
-
-{{< board-carousel >}}
-<br>
+- **Remote control**: You can use the Viam app, the Viam mobile app, or your own custom apps, to control and monitor your machines securely **from anywhere in the world**.
+- **Local control**: In factory settings or remote environments, you can operate and monitor your machines over a local area network (LAN).
 
 ## Viam Cloud
 
 The [Viam app](https://app.viam.com) is the ecosystem for configuring and managing machines which provides:
 
-- the user interface for configuring machines
-- a community registry of resources for machines
-- capabilities to make your machines better and smarter
-
-### Configuration
-
-As you configure your machine, you can test each resource to confirm it is working as expected in the same UI you use for configuration.
-
-{{<gif webm_src="/test.webm" mp4_src="/test.mp4" alt="Test a camera stream" max-width="600px" class="aligncenter">}}
-
-### Extensibility
+- **Configuration**: As you configure your machine, you can test each resource to confirm it is working. as expected in the same UI you use for configuration.
```

Comment:
This is confusing because we already had a "configuration" bullet above. Also typo.
```suggestion
- **Configuration and testing**: As you configure your machine, you can test each resource to confirm it is working as expected in the same UI you use for configuration.
```
or
```suggestion
- **Test your resources**: As you configure your machine, you can test each resource to confirm it is working as expected in the same UI you use for configuration.
```

Reply:
changed the above instead